### DISCUSSION TITLE
Error "Please enable module in Config.h" [RESOLVED]

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#956] [Need help]
Please help, after update to lasted version, I get error.

Error "Please enable module in Config.h"

CART READER V3
Version: 13.1
Hardware : HDW3 (physically my reader look's like HDW3)
Clock Generator that I got (red one) shows : HDW4/5 

![IMG_6809](https://github.com/sanni/cartreader/assets/38383423/d12f047d-f98c-48b0-916a-acf17b2528a2)
![68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f732f7772666676396c647a6675337176392f74656c6c6c687776657273696f6e2e6a7065673f646c3d31](https://github.com/sanni/cartreader/assets/38383423/baf55f3b-4713-439e-8f1d-622f11a089aa)
![68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f732f76646d62393764677a616d71637a372f74656c6c636c6f636b67656e2e6a7065673f646c3d31](https://github.com/sanni/cartreader/assets/38383423/a5a5c5e4-952a-483a-b94b-415628ca935d)

```
/*==== HARDWARE VERSION ===========================================*/

/*
    Choose your hardware version:
*/

//#define HW5
//#define HW4
#define HW3
//#define HW2
//#define HW1
//#define SERIAL_MONITOR

/****/

/*==== HARDWARE MODULES ===========================================*/

/* [ Automatic Voltage Selection ---------------------------------- ]
    Enable this if you have the VSELECT module.
*/

//#define ENABLE_VSELECT

/****/

/* [ Clock Generator ---------------------------------------------- ]
    Enable this if you have the clock generator module. This will
    automatically be enabled if you selected HW2 or newer above.
*/

#define clockgen_installed

/****/

/* [ Real Time Clock ---------------------------------------------- ]
    Enable this if you have the RTC module. You can configure the
    type later in this file.
*/

//#define RTC_installed

/****/

```

**Comment by PsyK0p4T** [comment]
Can you retry after enabling in config.h :

//#define ENABLE_FLASH
//#define ENABLE_FLASH16

Also, even if changes are not major since v12.1, try to update to the current master branch (dev version, currently 12.2)

**Reply by StanMOI** [reply]
> Can you retry after enabling in config.h :
> 
> //#define ENABLE_FLASH //#define ENABLE_FLASH16
> 
> Also, even if changes are not major since v12.1, try to update to the current master branch (dev version, currently 12.2)

Yeah!
It was it !
I enabled :

```
/* [ Flashrom Programmer for SNES repros -------------------------- ]
*/

#define enable_FLASH
#define enable_FLASH16

/****/
```
Where can I find this dev version?

**Reply by PsyK0p4T** [reply]
Cool !

You can download it from the project home page 
![image](https://github.com/sanni/cartreader/assets/87064902/7eb4c040-c049-45bf-8c38-f401c4603d30)

**Reply by StanMOI** [reply]
Thank you
